Source/WebCore/platform/graphics/PixelBufferConversion.cpp

```diff
@@ -290,7 +290,6 @@ static void convertImagePixelsUnaccelerated(const ConstPixelBufferConversionView
     }
 }
 
-#if !(USE(ACCELERATE) && USE(CG))
 static void copyImagePixels(const ConstPixelBufferConversionView& source, const PixelBufferConversionView& destination, const IntSize& destinationSize)
```

Comment:
Is this _faster_ than `convertImagePixelsAccelerated` for the cases where format and color space matches?

Reply:
Yeah, I'm also wondering the same question. I don't know the answer and how to measure that reliably.
This code was added by <https://commits.webkit.org/220895@main>.
https://github.com/WebKit/WebKit/blob/f19c746184e9553ed50a06403816f87bca59af55/Source/WebCore/platform/graphics/cg/ImageBufferCGBackend.cpp#L241-L244
It seems to be the author's intention.